pstatus-graphql-ktor/src/main/kotlin/gov/cdc/ocio/processingstatusapi/loaders/UploadStatsLoader.kt

```diff
@@ -72,6 +71,18 @@ class UploadStatsLoader: CosmosLoader() {
                         + ") r where r.totalCount > 1"
                 )
 
+        val unDeliveredUploadsQuery = (
```

Comment:
This query is sufficient for detecting explicitly undelivered uploads with reports that contain a failure status.  However, we need a second query to look for uploads where this report is missing.

Reply:
Updated the code to fetch the uploads where the report is missing for the stage, "blob-file-copy". For a given dataStreamId, dataStreamRoute and the start and the end dates, we check for any difference in the total uploads with stage-action = "metadata-verify" and stage-action ="blob-file-copy", get the respective undelivered upload ids as well.